performance_manager/tests/test_postgres.py

```diff
@@ -1,27 +1,73 @@
 import logging
 import os
-from typing import Iterator
+from functools import lru_cache
+from typing import Dict, Iterator, List, Optional, Sequence, Tuple, Union
 
+import pyarrow
 import pytest
+import sqlalchemy as sa
+from _pytest.monkeypatch import MonkeyPatch
+from pyarrow import fs, parquet
 
-from lib.postgres_utils import (
-    DatabaseManager,
-    get_unprocessed_files,
+from lib import process_static_tables
+from lib.l0_gtfs_rt_events import (
+    combine_events,
+    get_gtfs_rt_paths,
+    process_gtfs_rt_files,
+    upload_to_database,
 )
+from lib.l0_rt_trip_updates import (
+    get_and_unwrap_tu_dataframe,
+    join_tu_with_gtfs_static,
+    reduce_trip_updates,
+)
+from lib.l0_rt_vehicle_positions import (
+    get_vp_dataframe,
+    join_vp_with_gtfs_static,
+    transform_vp_datatypes,
+    transform_vp_timestamps,
+)
+from lib.postgres_schema import (
+    MetadataLog,
+    StaticCalendar,
+    StaticRoutes,
+    StaticStops,
+    StaticStopTimes,
+    StaticTrips,
+    VehicleEvents,
+)
+from lib.postgres_utils import DatabaseManager
 
 HERE = os.path.dirname(os.path.abspath(__file__))
+LAMP_DIR = os.path.join(HERE, "test_files", "lamp")
 
 
-@pytest.fixture(autouse=True)
-def set_environment_variables() -> Iterator[None]:
+@lru_cache
+def test_files() -> List[str]:
+    """
+    collaps all of the files in the lamp test files dir into a list
+    """
+    paths = []
+
+    for root, _, files in os.walk(LAMP_DIR):
+        for file in files:
+            if "DS_Store" in file:
+                continue
+            paths.append(os.path.join(root, file))
+
+    paths.sort()
+    return paths
```

Comment:
walks the test dir and turns it into a list. lru cache so we can just use it like a list.